wasm.md

```diff
@@ -147,16 +147,16 @@ The sorts `EmptyStmt` and `EmptyStmts` are administrative so that the empty list
     syntax EmptyStmt
  // ----------------
 
-    syntax Instr ::= EmptyStmt
-    syntax Defn  ::= EmptyStmt
+    syntax Instr ::= EmptyStmt [unused]
+    syntax Defn  ::= EmptyStmt [unused]
     syntax Stmt  ::= Instr | Defn
  // -----------------------------
 
-    syntax EmptyStmts ::= List{EmptyStmt , ""} [klabel(listStmt)]
-    syntax Instrs     ::= List{Instr     , ""} [klabel(listStmt)]
-    syntax Defns      ::= List{Defn      , ""} [klabel(listStmt)]
-    syntax Stmts      ::= List{Stmt      , ""} [klabel(listStmt)]
- // -------------------------------------------------------------
+    syntax EmptyStmts ::= List{EmptyStmt , ""} [klabel(listStmt), unused]
+    syntax Instrs     ::= List{Instr     , ""} [klabel(listStmt), unused]
+    syntax Defns      ::= List{Defn      , ""} [klabel(listStmt), unused]
+    syntax Stmts      ::= List{Stmt      , ""} [klabel(listStmt), unused]
+ // ---------------------------------------------------------------------
```

Comment:
Why do we need `unused` here? There are rules below that use these:

```
    syntax K ::= sequenceStmts  ( Stmts  ) [function]
               | sequenceDefns  ( Defns  ) [function]
               | sequenceInstrs ( Instrs ) [function]
 // -------------------------------------------------
    rule sequenceStmts(.Stmts) => .
    rule sequenceStmts(S SS  ) => S ~> sequenceStmts(SS)
    rule sequenceDefns(.Defns) => .
    rule sequenceDefns(D DS  ) => D ~> sequenceDefns(DS)
    rule sequenceInstrs(.Instrs) => .
    rule sequenceInstrs(I IS   ) => I ~> sequenceInstrs(IS)
```

@dwightguth is this expected?

Reply:
This definitely looks like it is used. Create an issue with a minimal example and assign it to me and I'll look into it.